### PR TITLE
Ds detect

### PR DESCRIPTION
**Pull request by Windwoes**
If the DS is also installed, show a dialog explaining the situation with options to uninstall either the RC or DS.

### CODE REVIEW

File: Inspection/build.gradle [hunk 1]
```diff
@@ -32,5 +32,5 @@ repositories {
 
 dependencies {
     compile project(':RobotCore')
-    compile project(':OpenFTC')
+    compile project(':openftc')
```
Comment (WardBenjamin) [inline Inspection/build.gradle]: Why did you make references to the module lowercase? This is inconsistent with the other modules in the project.

File: openftc/src/main/java/org/openftc/openftc/OpenFTCConfig.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.openftc;
+package org.openftc.openftc;
```
Comment (WardBenjamin) [inline openftc/src/main/java/org/openftc/openftc/OpenFTCConfig.java]: Why expand the package name? I get the change from com.openftc to org.openftc, but why add another layer?

**Review by WardBenjamin — COMMENTED** [review]
Looks mostly good

**Comment by Windwoes** [comment]
@WardBenjamin No reason, sorry about that. I was going to fix that, got distracted, and when I came back I had completely forgotten. I Added some commits to fix it.

**Pull request merged**